conformance/tests/httproute-response-header-modifier.go

```diff
@@ -168,48 +183,6 @@ var HTTPRouteResponseHeaderModifier = suite.ConformanceTest{
 			},
 			Backend:   "infra-backend-v1",
 			Namespace: ns,
-		}, {
-			Request: http.Request{
-				Path: "/response-and-request-header-modifiers",
-				Headers: map[string]string{
```

Comment:
why removing this?

Reply:
This conformance test is for asserting response headers modification but this test case includes a `RequestHeader` filter.  I'd say that's out of scope for the test. 

Do folks feel it necessary to keep this?

Reply:
Don't have a strong opinion on this but it's still left in `conformance/tests/httproute-response-header-modifier.yaml` atm.

Reply:
I've reverted the changes and split the test cases. Only a subset of them will run when it's a Backend Filter in order for the features not to mix.

Reply:
The idea was to test it along with the core feature to see that it still conformant if both are specified. (i.e to avoid a scenario where if responseHeaderModifier is specified, requestHeaderModifier is ignored. 

Why did you need to split the testcases ?

Reply:
Since the BackendRef features are Extended we can't guarantee implementatons implement both filters. So we can't have tests that mix them